### ISSUE TITLE
Navigating notification a11y view with alt+]/[ should announce where you are in the list

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#27721]
Testing -188528

I'm expecting something like "notification x of y", instead it just announces the next focused notification, even if it's the same one:

![image](https://github-com/microsoft/vscode/assets/2193314/929ae59f-a833-44c8-952e-7b69dc528c21)
